ajax/libs/unistore/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "unistore",
+  "filename": "dist/unistore.umd.js",
+  "version": "3.0.2",
+  "description": "Dead simple centralized state container (store) with preact and react bindings.",
+  "keywords": [
+    "preact",
+    "component",
+    "state machine",
+    "redux"
```

Comment:
Let's add more keyword, let developers can easier find this library.
![2017-12-27 1 07 30](https://user-images.githubusercontent.com/8349587/34371313-0d5e7a34-eb07-11e7-9007-40b12c3b749d.png)
Thanks